index.js

```diff
@@ -443,6 +443,18 @@ function makeEnv (data, opts, prefix, env) {
     }
   }
 
+  // assign script specific environnement variables (ignores non string values)
+  if (typeof opts.env === 'object') {
+    for (var k in opts.env) {
+      if (opts.env.hasOwnProperty(k)) {
```

Comment:
`.hasOwnProperty` should never be called directly off of objects; what about an env var named `hasOwnProperty` (which is entirely valid)? Always do this:
```suggestion
      if (Object.prototype.hasOwnProperty.call(opts.env, k)) {
```